resources/tpch_validation_queries.sql

```diff
@@ -0,0 +1,763 @@
+ *        WHERE
+ *            p_partkey = ps_partkey
+ *            AND s_suppkey = ps_suppkey
+ *            AND s_nationkey = n_nationkey
+ *            AND n_regionkey = r_regionkey
+ *            AND r_name = '[REGION]'
+ *        )
+ * ORDER BY s_acctbal DESC, n_name, s_name, p_partkey;
+ *
+ * Changes:
+ *  1. Random values are hardcoded
+ *  2. Changed to ordering in the FROM clause for better join ordering
+ */
+SELECT s_acctbal, s_name, n_name, p_partkey, p_mfgr, s_address, s_phone, s_comment FROM "part", partsupp, supplier, nation, region WHERE p_partkey = ps_partkey AND s_suppkey = ps_suppkey AND p_size = 15 AND p_type like '%BRASS' AND s_nationkey = n_nationkey AND n_regionkey = r_regionkey AND r_name = 'EUROPE' AND ps_supplycost = (SELECT min(ps_supplycost) FROM supplier, partsupp, nation, region WHERE p_partkey = ps_partkey AND s_suppkey = ps_suppkey AND s_nationkey = n_nationkey AND n_regionkey = r_regionkey AND r_name = 'EUROPE') ORDER BY s_acctbal DESC, n_name, s_name, p_partkey;
+
+
+/**
+ * TPC-H 3
+ *
+ * Original:
+ *
+ * SELECT l_orderkey, sum(l_extendedprice*(1-l_discount)) as revenue, o_orderdate, o_shippriority
+ * FROM customer, orders, lineitem
+ * WHERE c_mktsegment = '[SEGMENT]' AND c_custkey = o_custkey AND l_orderkey = o_orderkey
+ * AND o_orderdate < date '[DATE]' AND l_shipdate > date '[DATE]'
+ * GROUP BY l_orderkey, o_orderdate, o_shippriority
+ * ORDER BY revenue DESC, o_orderdate;
+ *
+ * Changes:
+ *  1. Random values are hardcoded
+ *  2. implicit type conversions for arithmetic operations are not supported
+ *    a. changed 1 to 1.0 explicitly
+ *  3. Be aware that we ignore the column ordering here.
+ */
+SELECT l_orderkey, SUM(l_extendedprice*(1.0-l_discount)) as revenue, o_orderdate, o_shippriority FROM customer, orders, lineitem WHERE c_mktsegment = 'BUILDING' AND c_custkey = o_custkey AND l_orderkey = o_orderkey AND o_orderdate < '1995-03-15' AND l_shipdate > '1995-03-15' GROUP BY l_orderkey, o_orderdate, o_shippriority ORDER BY revenue DESC, o_orderdate;
+
+
+/**
+ * TPC-H 4
+ *
+ * Original:
+ *
+ * SELECT
+ *    o_orderpriority,
+ *    count(*) as order_count
+ * FROM orders
+ * WHERE
+ *    o_orderdate >= date '[DATE]'
+ *    AND o_orderdate < date '[DATE]' + interval '3' month
+ *    AND exists (
+ *        SELECT *
+ *        FROM lineitem
+ *        WHERE
+ *            l_orderkey = o_orderkey
+ *            AND l_commitdate < l_receiptdate
+ *        )
+ * GROUP BY o_orderpriority
+ * ORDER BY o_orderpriority;
+ *
+ * Changes:
+ *  1. Random values are hardcoded
+ *  2. dates are not supported
+ *    a. use strings as data type for now
+ *    b. pre-calculate date operation
+ */
+SELECT o_orderpriority, count(*) as order_count FROM orders WHERE o_orderdate >= '1996-07-01' AND o_orderdate < '1996-10-01' AND exists (SELECT *FROM lineitem WHERE l_orderkey = o_orderkey AND l_commitdate < l_receiptdate) GROUP BY o_orderpriority ORDER BY o_orderpriority;
+
+
+/**
```

Comment:
can you make these sql-style comments so that this file is valid sql?

Reply:
Haha, now we have two conflicting ideas on what this should look like. I seem to be missing in advantage of having this as a sql file.

Reply:
![image](https://user-images.githubusercontent.com/2679018/52118578-f864c380-2616-11e9-8d64-70181396c3af.png)

I thought we consented on an SQL file? I do not have strong opinions here, but as a quick resource for c/p a non-cpp file seems lees intimidating, no?

Reply:
Ah, ok. I understood this as a file with SQL commands, not necessarily an .sql file. In the end, it probably doesn't make that much of a difference anyway.

Reply:
> can you make these c++-comments valid sql-style comments so that this file is valid sql?

Done. Still, we can't just load the file via `script resources/tpch_validation_queries.sql` in the console because there seems to be some newline handling issue.

Reply:
Open issue over at hyrise/sql-parser?

Reply:
I don't know whether this is a parser issue or a problem with the console. I will open an issue after finding out.

Reply:
This fails:
```
-- Comment

SELECT * FROM nation;
```

while this works:
```
-- Comment
SELECT * FROM nation;
```

I don't know if this is the intended parser bevaiour. I understand that blank lines in a statement are a problem. Blank lines between statements should not in my opinion.